packages/providers/mgt-teamsfx-provider/src/TeamsFxProvider.ts

```diff
@@ -179,4 +108,18 @@ export class TeamsFxProvider extends IProvider {
     this._accessToken = token ?? (await this.getAccessToken());
     this.setState(this._accessToken ? ProviderState.SignedIn : ProviderState.SignedOut);
   }
+
+  /**
+   * Update scopes
+   *
+   * @param {string[]} scopes
+   * @memberof TeamsFxProvider
+   */
+  public updateScopes(scopes: string[]) {
```

Comment:
After the initial scopes are set, what happens if we update with another set of scopes? It would go through another consent flow?

Reply:
As far as I know, if updated scopes is subset of original scopes, then it will works fine without any error.
However, if updated scopes is superset of original scopes, then when call getToken, it will throw error that "need consent".

Update scopes is not very common, I found this function from your original code and not sure whether it is necessary, if it not necessary, we can just remove function to avoid confusing for users,  what do you think about it?

Reply:
I remove this function to avoid confusing